portal/src/main/webapp/WEB-INF/web.xml

```diff
@@ -134,6 +134,20 @@
     </servlet-mapping>
 
     <servlet>
+        <servlet-name>rest</servlet-name>
+        <servlet-class>org.springframework.web.servlet.DispatcherServlet</servlet-class>
+        <init-param>
+            <param-name>contextConfigLocation</param-name>
+            <param-value>classpath:applicationContext-rest.xml</param-value>
+        </init-param>
+        <load-on-startup>1</load-on-startup>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>rest</servlet-name>
```

Comment:
As noted, we will do away with rest root endpoint when code is merged from api-design to rc.